R/plot2.R

```diff
@@ -90,20 +90,30 @@
 #'    specifying "none").
 #'    - A list or, equivalently, a dedicated `legend()` function with supported
 #'    legend arguments, e.g. "bty", "horiz", and so forth.
+#' @param col plotting color. Character, integer, or vector of length equal to
+#'   the number of categories in the `by` variable. See `col`. Specifying colors
+#'   manually in `plot2` should not be necessary unless users wish to override
+#'   the automatic colors produced by the grouping process. Typically, this
+#'   would only be done if grouping characterists are deferred to some other
+#'   graphical parameter (i.e., passing the "by" keyword to one of `pch`, `lty`,
+#'   or `bg`; see below.)
 #' @param pch plotting "character", i.e., symbol to use. Character, integer, or
 #'   vector of length equal to the number of categories in the `by` variable.
 #'   See `pch`. In addition, users can supply a special `pch = "by"` convenience
 #'   argument, in which case the characters will automatically loop over the
 #'   number groups. This automatic looping will begin at the global character
 #'   value (i.e., `par("pch")`) and recycle as necessary.
-#' @param col plotting color. Character, integer, or vector of length equal to
-#'   the number of categories in the `by` variable. See `col`.
 #' @param lty line type. Character, integer, or vector of length equal to the
 #'   number of categories in the `by` variable. See `lty`. In addition, users
 #'   can supply a special `lty = "by"` convenience argument, in which case the
 #'   line type will automatically loop over the number groups. This automatic
 #'   looping will begin at the global line type value (i.e., `par("lty")`) and
 #'   recycle as necessary.
+#' @param bg background (fill) color for the open plot symbols 21:25: see
+#'   `points.default`. In addition, users can supply a special `bg = "by"`
+#'   convenience argument, in which case the background color will inherit the
+#'   automatic group coloring intended for the `col` parameter.
+#' @param cex character expansion.
```

Comment:
docstring could be more explicit. Here is what `?par` says: a numerical vector giving the amount by which plotting characters and symbols should be scaled relative to the default. This works as a multiple of par. NULL and NA are equivalent to 1.0. Note that this does not affect annotation: see below.